data/data_0DD0A0_1.s

```diff
@@ -120,146 +120,187 @@ glabel D_800DCBB4
 .endm
 
 glabel D_800DCC08
+# Mario Raceway Bomb Kart spawns
```

Comment:
Why did they put this data here? This should be in the course_data files.

Reply:
Who knows. Maybe the bomb karts were a last minute addition? Could explain why there's an unused bomb kart portrait floating around in the data